src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapSeparatorAndInterfaces.java

```diff
@@ -1,6 +1,6 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
+package com.puppycrawl.tools.checkstyle.checks.whitespace.operatorwrap;
```

Comment:
How can we move an input file and not have to change any tests?
If a test was using this file, it would be complaining about it not being able to find it.
If no tests are failing from missing this file, then it was never used by any tests.
Please confirm and delete file if so.